templates/web/base/waste/bulky/confirmation.html

```diff
@@ -1,35 +1,31 @@
+[%
+IF report.category == 'Bulky collection';
+    title = 'Bulky collection booking confirmed';
+ELSIF report.category == 'Small items collection';
+    title = 'Small items collection booking confirmed';
+ELSE;
+    title = 'Enquiry has been submitted';
+END ~%]
+
 [% PROCESS 'waste/header.html' %]
 
-<div class="govuk-warning-text whole-page govuk-!-margin-bottom-9">
-    <div class="govuk-warning-text__img govuk-!-margin-bottom-5">
-       <span class="govuk-warning-text__icon success" aria-hidden="true">&#x2714;</span>
-    </div>
-    <div class="govuk-warning-text__content">
-        <span class="govuk-warning-text__assistive">Important information</span>
-        <h3 class="govuk-heading-l govuk-warning-text__heading">Collection booked</h3>
-        <dl>
-            <dt><strong>Booking reference number</strong></dt>
-            <dd class="govuk-!-margin-bottom-4">[% report.id %]</dd>
-            <dt><strong>Date of collection</strong></dt>
-            <dd class="govuk-!-margin-bottom-6">
-                <p class="govuk-!-margin-bottom-1">[% cobrand.bulky_nice_collection_date(report) %]</p>
-                <small style="max-width: 350px; display:block;">
-                    Item collection starts from [% cobrand.bulky_nice_collection_time %].
-                    Please have your items ready and dismantled if required.
-                </small>
-            </dd>
-          [% IF reference %]
-            <dt><strong>Payment reference</strong></dt>
-            <dd class="govuk-!-margin-bottom-4">[% reference %]</dd>
```

Comment:
We should still include the payment reference (see e.g. waste/garden/subscribe_confirm.html).